app/services/plans/update_service.rb

```diff
@@ -110,9 +110,11 @@ def process_charges(plan, params_charges)
             invoiceable = payload_charge.delete(:invoiceable)
             min_amount_cents = payload_charge.delete(:min_amount_cents)
             tax_codes = payload_charge.delete(:tax_codes)
+            properties = payload_charge.delete(:properties)
 
             charge.invoiceable = invoiceable if License.premium? && !invoiceable.nil?
             charge.min_amount_cents = min_amount_cents || 0 if License.premium?
+            charge.properties = properties.presence || Charges::BuildDefaultPropertiesService.call(payload_charge[:charge_model])
```

Comment:
Layout/LineLength: Line is too long. [129/120] (https://rubystyle.guide#max-line-length)